__tests__/__unit__/extension.unit.test.ts

```diff
@@ -1033,99 +894,357 @@ describe("Extension Unit Tests", () => {
+                value: jest.fn(() => {
+                    return {
+                        allProfiles: [{name: "firstName"}, {name: "secondName"}],
+                        defaultProfile: {name: "firstName"},
+                        listProfile: jest.fn(()=>{
+                            return {};
+                        }),
+                        loadNamedProfile: mockLoadNamedProfile
+                    };
+                })
+            });
+
+            // Assert edge condition user cancels the input path box
+            createQuickPick.mockReturnValue({
+                placeholder: "Choose \"Create new...\" to define a new profile or select an existing profile to Add to the Data Set Explorer",
+                activeItems: [qpItem],
+                ignoreFocusOut: true,
+                items: [qpItem],
+                value: entered,
+                show: jest.fn(()=>{
+                    return {};
+                }),
+                hide: jest.fn(()=>{
+                    return {};
+                }),
+                onDidAccept: jest.fn(()=>{
+                    return {};
+                })
+            });
+
+            await extension.addZoweSession(testTree);
+            expect(extension.addZoweSession).toHaveBeenCalled();
+
+        });
+
+        it("Testing that addSession if listProfile is invalid", async () => {
+            const entered = "fake";
+            const addSession = jest.spyOn(extension, "addZoweSession");
+
+            Object.defineProperty(Profiles, "getInstance", {
+                value: jest.fn(() => {
+                    return {
+                        allProfiles: [{name: "firstName"}, {name: "secondName"}],
+                        defaultProfile: {name: "firstName"},
+                        createNewConnection: jest.fn(()=>{
+                            return {};
+                        }),
+                        loadNamedProfile: mockLoadNamedProfile
+                    };
+                })
+            });
+
+            // Assert edge condition user cancels the input path box
+            createQuickPick.mockReturnValue({
+                placeholder: "Choose \"Create new...\" to define a new profile or select an existing profile to Add to the Data Set Explorer",
+                activeItems: [qpItem],
+                ignoreFocusOut: true,
+                items: [qpItem],
+                value: entered,
+                show: jest.fn(()=>{
+                    return {};
+                }),
+                hide: jest.fn(()=>{
+                    return {};
+                }),
+                onDidAccept: jest.fn(()=>{
+                    return {};
+                })
+            });
+
+            await extension.addZoweSession(testTree);
+            expect(extension.addZoweSession).toHaveBeenCalled();
+
+        });
+    });
+
+    it("Testing that createFile is executed successfully", async () => {
```

Comment:
Shouldn't all `dsActions.createFile()` tests be removed from `extension.unit.test.ts`, if the tests have been refactored?